app/services/personal_key_generator.rb

```diff
@@ -29,19 +29,19 @@ def verify(plaintext_code)
     Devise.secure_compare(encrypted_code, user_access_key.encrypted_password)
   end
 
-  private
-
-  attr_reader :length, :user, :key_maker
-
   def normalized_code(plaintext_code)
```

Comment:
why not moved `normalized_code` into `PersonalKeyValidator`? afaik it's the main place it is used.